### DISCUSSION TITLE
Make Frontend Shit Again

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#1085] [Dicas]
:heart_eyes: 

https://makefrontendshitagain.party/

**Comment by didiraja** [comment]
haiuhaiuahiuah que incrível, lembraram até do botão direito bloqueado

**Comment by felipefialho** [comment]
Sdds, haahaha

**Comment by mateusmiguel** [comment]
tem que dar play na musiquinha pra experiencia ser completa
